src/wast-parser.cc

```diff
@@ -1545,13 +1602,23 @@ Result WastParser::ParseConst(Const* const_) {
       result = ParseDouble(literal.type, s, end, &const_->f64_bits);
       break;
 
+    case Opcode::V128Const:
+      const_->type = Type::V128;
```

Comment:
We only want to allow parsing this if the SIMD feature is enabled, see other examples in this file using `ErrorUnlessOpcodeEnabled`.

Reply:
Agree.